processor/abci.go

```diff
@@ -48,13 +48,17 @@ import (
 
 var (
 	ErrPublicKeyCannotSubmitTransactionWithNoBalance  = errors.New("public key cannot submit transaction without balance")
+	ErrUnexpectedTxPubKey                             = errors.New("no one listens to the public keys that signed this oracle data")
 	ErrTradingDisabled                                = errors.New("trading disabled")
 	ErrNoTransactionAllowedDuringBootstrap            = errors.New("no transaction allowed during the bootstraping period")
 	ErrMarketProposalDisabled                         = errors.New("market proposal disabled")
 	ErrAssetProposalDisabled                          = errors.New("asset proposal disabled")
 	ErrNonValidatorTransactionDisabledDuringBootstrap = errors.New("non validator transaction disabled during bootstrap")
 	ErrCheckpointRestoreDisabledDuringBootstrap       = errors.New("checkpoint restore disabled during bootstrap")
 	ErrAwaitingCheckpointRestore                      = errors.New("transactions not allowed while waiting for checkpoint restore")
+	ErrOracleDataNormalization                        = func(err error) error {
+		return fmt.Errorf("error normalizing incoming oracle data: %s", err.Error())
```

Comment:
You can use `%w`, it wraps the underlaying error and works better with error API `errors.Is()`

```suggestion
		return fmt.Errorf("error normalizing incoming oracle data: %w", err)
```